cmd/controller/controller.go

```diff
@@ -45,24 +47,30 @@ func init() {
 	if importerImage == "" {
 		importerImage = IMPORTER_DEFAULT_IMAGE
 	}
-	pullPolicy = IMPORTER_DEFAULT_PULL_POLICY
-	if pp := os.Getenv(IMPORTER_PULL_POLICY); len(pp) != 0 {
-		pullPolicy = pp
+	
+	clonerImage = os.Getenv(CLONER_IMAGE)
+	if clonerImage == "" {
+		clonerImage = CLONER_DEFAULT_IMAGE
 	}
-
+	
+	PullPolicy = DEFAULT_PULL_POLICY
+	if pp := os.Getenv(PULL_POLICY); len(pp) != 0 {
+		PullPolicy = pp
+	}
+	
 	// get the verbose level so it can be passed to the importer pod
-	defVerbose := fmt.Sprintf("%d", IMPORTER_DEFAULT_VERBOSE) // note flag values are strings
-	verbose = defVerbose
+	defVerbose := fmt.Sprintf("%d", DEFAULT_VERBOSE) // note flag values are strings
+	Verbose = defVerbose
 	// visit actual flags passed in and if passed check -v and set verbose
 	flag.Visit(func(f *flag.Flag) {
 		if f.Name == "v" {
-			verbose = f.Value.String()
+			Verbose = f.Value.String()
 		}
 	})
-	if verbose == defVerbose {
+	if Verbose == defVerbose {
 		glog.V(Vuser).Infof("Note: increase the -v level in the controller deployment for more detailed logging, eg. -v=%d or -v=%d\n", Vadmin, Vdebug)
 	}
-
+	
```

Comment:
nit: leading blanks should be deleted. Best to run `gofmt -w -s` on all changed files.